cspell.json

```diff
@@ -30,6 +30,7 @@
 		"isbot",
 		"Jérémie",
 		"knip",
+		"neurodiversity",
```

Comment:
We should remove this once/if https://github.com/streetsidesoftware/cspell-dicts/pull/4293 is accepted and merged/released.